src/OXRS_LCD.cpp

```diff
@@ -178,6 +260,27 @@ void OXRS_LCD::update(void)
       _last_lcd_trigger = 0L;
     }
   }
+  
+  // turn of rx LED if timed out
```

Comment:
Type `turn of`

Reply:
"Type turn of" ?  ;-) typo in typo correction ;-)
done